deploy/controller/operator.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: multicluster-integrations
+  namespace: open-cluster-management
+  labels:
+    name: multicluster-integrations
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: multicluster-integrations
+  template:
+    metadata:
+      labels:
+        name: multicluster-integrations
+    spec:
+      serviceAccountName: multicluster-operators
+      volumes:
+      - name: multicluster-integrations-syncresource 
+        emptyDir: {}
+      containers:
+      - name: multicluster-integrations-syncresource
+        image: quay.io/open-cluster-management/multicloud-integrations:latest
+        imagePullPolicy: Always
+        command:
+        - /usr/local/bin/gitopssyncresc
+        - --appset-resource-dir=/etc/gitops-resources
+        env:
+        - name: WATCH_NAMESPACE
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: DEPLOYMENT_LABEL
+          value: multicluster-integrations-syncresource
+        - name: OPERATOR_NAME
+          value: multicluster-integrations
+        - name: ACCESS_TOKEN
+          value: "sha256~TWYxOVynSaSGSC2NBclceQvzWT7DGqSmolEc7NtXhyE"          
```

Comment:
Since the `ACCESS_TOKEN` depends on the cluster it's accessing, this would require changes to the operator.yaml before being deployed, is there a workaround or is it fine like this?

Reply:
Yeah, this env var was as a workaround before and it should be removed now.

Reply:
@philipwu08 Thanks for spotting that. We actually don't need this token anymore. It was a temp workaround when we had a problem with search but that problem has since been resolved. Let me remove it. Thanks.